### PR TITLE
Async await b

### PR DESCRIPTION
**Pull request by MatthewBerglund**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <blockquote>
           <p id="quote"></p>
         </blockquote>
-        <figcaption>— <cite id="ref"></cite><span id="ref-year"></span></figcaption>
+        <figcaption>— <cite id="movie"></cite><span id="year"></span></figcaption>
```
Comment (motine) [inline index.html:20]: nice one. much clearer.

File: main.js [hunk 2]
```diff
@@ -51,6 +44,18 @@ function encodeTweetForURI(tweetText) {
   return encodedTweet;
 }
 
+async function fetchQuotes() {
```
Comment (motine) [inline main.js:47]: how do you like async? I personally think it makes the reading flow nicer. Especially in combination of multiple promises (e.g. fetch-then-json)

**Review by motine — APPROVED** [review]
nice work. I think this was a good exercise.

File: main.js (hunk 2 above)
Comment (MatthewBerglund) [reply main.js:47]: Yeah I like it a lot better than the Fetch API too for the same reason. As long as browser support isn't too much of an issue, I'd prefer to keep it.

Comment (motine) [reply main.js:47]: yeah. i think for most intents and purposes browser support is fine. let me know when you have more to review or when you want to have a chat again. merry xmas again!

**Pull request merged**